pkg/server/domain/service/webhook.go

```diff
@@ -187,28 +189,96 @@ func (c *webhookServiceImpl) patchComponentProperties(ctx context.Context, compo
 }
 
 func (c *customHandlerImpl) handle(ctx context.Context, webhookTrigger *model.ApplicationTrigger, app *model.Application) (interface{}, error) {
-	for comp, properties := range c.req.Upgrade {
-		component := &model.ApplicationComponent{
+	workflow := &model.Workflow{
+		AppPrimaryKey: webhookTrigger.AppPrimaryKey,
+		Name:          webhookTrigger.WorkflowName,
+	}
+	if err := c.w.Store.Get(ctx, workflow); err != nil {
+		return nil, err
+	}
+	switch c.req.Action {
+	case "approve":
```

Comment:
Consider to made them consts.

Reply:
Sure, will do that.

Reply:
done!

Reply:
I think you misunderstood me. I mean turn "approve" like string to consts:
```go
const (
  ActionApprove = "approve" 
  ...
)
```
In other place you could ref to them.

Reply:
Oh! Sorry I misunderstood it. I will add these changes.

Reply:
Now I think I have made all the strings into const.